src/components/SongHistory.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { faHistory } from '@fortawesome/free-solid-svg-icons';
+
+class SongHistory extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayList: false
+    };
+  }
+
+  toggleDisplay = () => {
+    this.setState({
+      displayList: !this.state.displayList
+    });
+  };
+
+  render() {
+    const { songHistory } = this.props;
+    const songs = songHistory.map(song => song.song).reverse();
+    return (
+      <button id='recent-song-history' onClick={this.toggleDisplay}>
+        {this.state.displayList && (
+          <div id='recent-song-list'>
+            {songs.map(song => (
+              <div className='recent-song-info' key={song.id}>
+                <img alt={`Album Title: ${song.album}`} src={song.art} />

```

Comment:
I'm being a bit nit-picking, but since the markup elements already provide the information, I think the image is purely decorative and can be hidden from screen readers.

```suggestion
                <img role="presentation" alt="" src={song.art} />
```

But it isn't that big of a deal. We can just leave it as is.

Reply:
Thanks, I addressed your suggestions in separate commits